bevy_rapier2d/examples/testbed2.rs

```diff
@@ -62,7 +62,9 @@ fn main() {
     app.init_resource::<ExamplesRes>()
         .add_plugins((
             DefaultPlugins,
-            EguiPlugin,
+            EguiPlugin {
+                enable_multipass_for_primary_context: false,
```

Comment:
Should this be `true` instead or is `false` the "sane default" for this feature?